src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -3077,6 +3078,13 @@ void ReportPurchasedAir(EnergyPlusData &state, int const PurchAirNum)
     PurchAir.HtRecTotHeatRate = PurchAir.HtRecSenHeatRate + PurchAir.HtRecLatHeatRate;
     PurchAir.HtRecTotCoolRate = PurchAir.HtRecSenCoolRate + PurchAir.HtRecLatCoolRate;
 
+    Real64 heatFuelEffValue = PurchAir.heatFuelEffSched->getCurrentVal();
+    PurchAir.ZoneTotHeatFuelRate = PurchAir.TotHeatRate / heatFuelEffValue;
+    PurchAir.TotHeatFuelRate = PurchAir.ZoneTotHeatRate / heatFuelEffValue;
+    Real64 coolFuelEffValue = PurchAir.heatFuelEffSched->getCurrentVal();
+    PurchAir.ZoneTotCoolFuelRate = PurchAir.ZoneTotCoolRate / coolFuelEffValue;
+    PurchAir.TotCoolFuelRate = PurchAir.TotCoolRate / coolFuelEffValue;
+
```

Comment:
Calculates Fuel Energy Rates from the Load and user-specified fuel efficiency values.